internal/handlers/pg/pgdb/pool.go

```diff
@@ -359,7 +403,12 @@ func (pgPool *Pool) CreateTableIfNotExist(ctx context.Context, db, collection st
 		return false, lazyerrors.Error(err)
 	}
 
-	if err := pgPool.CreateTable(ctx, db, collection); err != nil {
+	table, err := pgPool.GetTableName(ctx, db, collection)
```

Comment:
Again, there is a moment between `GetTableName` and `GetTableName` and we can't guarantee that during that moment the table won't be created. 

I believe this is a relatively minor thing, but I wonder how it would work with our hashes? E.g. is it possible that we can create two tables with different hashes for the same name?

Reply:
Hashes must be stable, they don't depend on anything else just on the passed string itself.